app/uk/gov/hmrc/trackingconsentfrontend/assets/src/userPreferenceFactory.ts

```diff
@@ -0,0 +1,16 @@
+// @ts-ignore
+import Cookies from 'js-cookie'
+
+const userPreferenceFactory = () => ({
+  userAcceptsAll: () => {
+    Cookies.set('userConsent', JSON.stringify({
+      version: '2020-01-01',
```

Comment:
Is 2020-01-01 the right value? It is March at the moment :)

Reply:
I'm happy either way.  I did change the interface slightly from the original `2020-01-01` so I'll update to `2020-03-01`.